creators_custom/RoutesCreatorFenix.py

```diff
@@ -0,0 +1,501 @@
+# coding=utf-8
+
+import os
+import sys
+import pickle
+from collections import OrderedDict
+from datetime import timedelta, datetime
+import overpy
+from osmapi import OsmApi
+from creators_default.RoutesCreator import RoutesCreator
+import creators_custom
+
+CHECK_FOR_NON_PLATFORMS = False
+
+class RoutesCreatorFenix(RoutesCreator):
+
+    def __init__(self, config):
+        self.config = config
+        creators_custom.ROUTES = self.get_routes("bus")
+
+    def __repr__(self):
+        rep = ""
+        if self.config is not None:
+            rep += str(self.config) + " | "
+        return rep
+
+    def add_routes_to_schedule(self, schedule):
+
+        routes = creators_custom.ROUTES
+
+        blacklist = ['10200', '12400', '328', '466', '665']
+        # Try to find OSM routes in Fenix data
+        for route_ref, route in sorted(routes.iteritems()):
+            found = False
+            for linha in linhas:
```

Comment:
A good example what would be different in Fenix from a default implementation (#26) :smile:
I moved the linhas.json (with schedule information) from the TripsCreator to the RoutesCreator. Sorry, it's very anoying I can't test this myself because of constant errors:
`overpy.exception.OverpassTooManyRequests: Too many requests`